test/Templates.Test/MvcTemplateTest.cs

```diff
@@ -86,6 +86,10 @@ private void MvcTemplate_IndividualAuthImpl(string targetFrameworkOverride, bool
             Assert.Contains("Microsoft.VisualStudio.Web.CodeGeneration.Design", projectFileContents);
             Assert.Contains("Microsoft.EntityFrameworkCore.Tools.DotNet", projectFileContents);
 
+            RunDotNetEfCreateMigration("mvc");
+
+            AssertEmptyMigration("mvc");
+
```

Comment:
Should we also try to do dotnet ef database update here to ensure that the migration is applied correctly?

Reply:
Probably not needed here if the generated migration is really empty.